src/app/ApplicationTemplate.UWP/Views/Content/Settings/SettingsPage.xaml

```diff
@@ -15,88 +14,108 @@
 			<RowDefinition Height="*" />
 		</Grid.RowDefinitions>
 
+		<!-- CommandBar -->
 		<CommandBar Content="Profile"
 					x:Uid="Settings_CommandBar" />
 
 		<ScrollViewer Grid.Row="1">
-
 			<StackPanel>
+
+				<!-- DataLoaderView -->
 				<dl:DataLoaderView Source="{Binding UserProfile}"
 								   MinHeight="270">
 					<DataTemplate>
-						<StackPanel Padding="24">
-							<!-- Name and Lastname -->
-							<TextBlock Style="{StaticResource Headline5}">
-									<Run Text="{Binding Data.FirstName}" /><Run
-											Text=" " /><Run
-											Text="{Binding Data.LastName}" />
-							</TextBlock>
-
-							<!-- Email -->
-							<TextBlock Text="Email"
-										x:Uid="Settings_Email"
-										Margin="0,15,0,0"
-										Style="{StaticResource Subtitle2}" />
-							<TextBlock Text="{Binding Data.Email}"
-										Style="{StaticResource Body2}" />
-
-							<Rectangle Margin="0,15"
-										Height="1"
-										HorizontalAlignment="Stretch"
-										Fill="{StaticResource MaterialOnSurfacePressedBrush}" />
-
-							<!-- Password -->
-							<TextBlock Text="Password"
-										x:Uid="Settings_Password"
-										Style="{StaticResource Subtitle2}" />
-							<TextBlock Text="xxxxxxxx"
-										Style="{StaticResource Body2}" />
-
-							<!-- Change password -->
-							<Button Content="CHANGE PASSWORD"
-									x:Uid="Settings_ChangePassword"
-									Margin="0,15,0,0"
-									HorizontalAlignment="Left"
-									Command="{Binding Parent.NavigateToResetPasswordPage}" />
+						<StackPanel>
+
+							<!-- User Info Section -->
+							<StackPanel Padding="24,24,24,0">
+
+								<!-- Name -->
+								<TextBlock Style="{StaticResource Headline5}">
+									<Run Text="{Binding Data.FirstName}" /><Run Text=" " /><Run Text="{Binding Data.LastName}" />
+								</TextBlock>
+
+								<!-- Email Label -->
+								<TextBlock Text="Email"
+									   x:Uid="Settings_Email"
+									   Style="{StaticResource Subtitle2}"
+									   Margin="0,15,0,0" />
+
+								<!-- Email Value -->
+								<TextBlock Text="{Binding Data.Email}"
+										   Style="{StaticResource Body2}" />
+							</StackPanel>
+							
+							<!-- Separator -->
+							<Rectangle Fill="{StaticResource MaterialOnSurfacePressedBrush}"
```

Comment:
I don't think it makes sense to use a "Pressed" brush for separators. I think we could create a brush specific to separator.